test_communication/test/test_requester_replier.py.in

```diff
@@ -25,6 +25,7 @@ def test_requester_replier():
         replier_rmw_env_key = 'RCL_ASSERT_RMW_ID_MATCHES'
     replier_env = dict(os.environ)
     replier_env[replier_rmw_env_key] = '@REPLIER_RMW@'
```

Comment:
Maybe it should always be just `RCL_ASSERT_RMW_ID_MATCHES`, as it is for C++. Applies to the requester too and maybe the pub/sub equivalents.